Js/form.js

```diff
@@ -1,20 +1,53 @@
+let savedData = {};
+const nameBox = document.getElementById('name');
+const textBox = document.getElementById('test');
 const email = document.getElementById('email');
 const form = document.getElementsByClassName('form')[0];
 const errorMessage = document.getElementsByClassName('error-message')[0];
 const warning = document.querySelector('.warning-icon');
+if (localStorage.getItem('user') != null) {
+  savedData = JSON.parse(localStorage.getItem('user'));
+  nameBox.value = savedData.fullName;
+  email.value = savedData.email;
+  textBox.value = savedData.textarea;
+}
+
+nameBox.addEventListener('focusout', () => {
+  const formData = {
+    fullName: nameBox.value,
+    textarea: textBox.value,
+    email: email.value,
+  };
+  localStorage.setItem('user', JSON.stringify(formData));
+});
+email.addEventListener('focusout', () => {
+  const formData = {
+    fullName: nameBox.value,
+    textarea: textBox.value,
+    email: email.value,
+  };
+  localStorage.setItem('user', JSON.stringify(formData));
+});
+textBox.addEventListener('focusout', () => {
+  const formData = {
+    fullName: nameBox.value,
+    textarea: textBox.value,
+    email: email.value,
+  };
+  localStorage.setItem('user', JSON.stringify(formData));
+});
```

Comment:
- [OPTIONAL] Great job here 💯 but I suggest Instead of adding `focusout` event on each input and making three functions, You can add `change` event on the Form to avoid repeating the same lines 😉
> For Example 
```
form.addEventListener('change', () => {
  const formData = {
    fullName: nameBox.value,
    textarea: textBox.value,
    email: email.value,
  };
  localStorage.setItem('user', JSON.stringify(formData));
});
```